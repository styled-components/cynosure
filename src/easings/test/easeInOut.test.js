// @flow
import easeInOut from '../easeInOut'

describe('easeInOut', () => {
  it('should return easeInOutBack cubic-bezier', () => {
    expect({
      'transition-timing-function': easeInOut('back'),
    }).toMatchSnapshot()
  })

  it('should return easeInOutCirc cubic-bezier', () => {
    expect({
      'transition-timing-function': easeInOut('circ'),
    }).toMatchSnapshot()
  })

  it('should return easeInOutCubic cubic-bezier', () => {
    expect({
      'transition-timing-function': easeInOut('cubic'),
    }).toMatchSnapshot()
  })

  it('should return easeInOutExpo cubic-bezier', () => {
    expect({
      'transition-timing-function': easeInOut('expo'),
    }).toMatchSnapshot()
  })

  it('should return easeInOutQuad cubic-bezier', () => {
    expect({
      'transition-timing-function': easeInOut('quad'),
    }).toMatchSnapshot()
  })

  it('should return easeInOutQuart cubic-bezier', () => {
    expect({
      'transition-timing-function': easeInOut('quart'),
    }).toMatchSnapshot()
  })

  it('should return easeInOutQuint cubic-bezier', () => {
    expect({
      'transition-timing-function': easeInOut('quint'),
    }).toMatchSnapshot()
  })
})
