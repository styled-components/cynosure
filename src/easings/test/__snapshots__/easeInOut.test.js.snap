// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`easeInOut should return easeInOutBack cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.680, -0.550, 0.265, 1.550)",
}
`;

exports[`easeInOut should return easeInOutCirc cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.785,  0.135, 0.150, 0.860)",
}
`;

exports[`easeInOut should return easeInOutCubic cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.645,  0.045, 0.355, 1.000)",
}
`;

exports[`easeInOut should return easeInOutExpo cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(1.000,  0.000, 0.000, 1.000)",
}
`;

exports[`easeInOut should return easeInOutQuad cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.455,  0.030, 0.515, 0.955)",
}
`;

exports[`easeInOut should return easeInOutQuart cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.770,  0.000, 0.175, 1.000)",
}
`;

exports[`easeInOut should return easeInOutQuint cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.860,  0.000, 0.070, 1.000)",
}
`;
